watershed_workflow/hydrography.py

```diff
@@ -633,8 +655,9 @@ def filter_small_rivers(rivers, count):
     return new_rivers
 
 
-def merge(river, tol=_tol):
-    """Remove inner branches that are short, combining branchpoints as needed."""
+def merge_old(river, tol=_tol):
```

Comment:
remove merge_old()?  Is it used now?

Reply:
no, I kept it just in case, but the updated merge so far has not given problems.